docker/addons/vault/scripts/vault_copy_certs.sh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/bash
+# Copyright (c) Abstract Machines
+# SPDX-License-Identifier: Apache-2.0
+
+set -euo pipefail
+
+scriptdir="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+
+# default env file path
+env_file="docker/.env"
+
+# default certs copy path
+certs_copy_path="docker/ssl/certs/"
+
+while [[ "$#" -gt 0 ]]; do
+    case $1 in
+        --env-file)
+            if [[ -z "${2:-}" ]]; then
+                echo "Error: --env-file requires a non-empty option argument."
+                exit 1
+            fi
+            env_file="$2"
+            if [[ ! -f "$env_file" ]]; then
+                echo "Error: .env file not found at $env_file"
+                exit 1
+            fi
+            shift
+            ;;
+        --certs-copy-path)
```

Comment:
For the below command i get error
```bash
docker/addons/vault/scripts/vault_copy_env.sh --env-file docker/.env --certs-copy-path ~/mainflux/
```

Error Message
```log 
Unknown parameter passed: --certs-copy-path
```

Reply:
@arvindh123 I am not able to recreate this error

``` 
$docker/addons/vault/scripts/vault_copy_certs.sh --env-file docker/.env --certs-copy-path certs/pki     
Copying certificate files to certs/pki
'/home/jeff/Documents/magistrala/docker/addons/vault/scripts/data/localhost.crt' -> 'certs/pkimagistrala-server.crt'
'/home/jeff/Documents/magistrala/docker/addons/vault/scripts/data/localhost.key' -> 'certs/pkimagistrala-server.key'
'/home/jeff/Documents/magistrala/docker/addons/vault/scripts/data/mg_int.key' -> 'certs/pkica.key'
'/home/jeff/Documents/magistrala/docker/addons/vault/scripts/data/mg_int_bundle.crt' -> 'certs/pkica.crt'
```

Reply:
@arvindh123 `vault_copy_env.sh` only accepts `--env-file` argument
To pass both `--env-file` and `--certs-copy-path` argument, run `vault_copy_certs.sh`